db/seeds/muscle_descriptions.rb

```diff
@@ -0,0 +1,77 @@
+
+  The calves are a very stubborn muscle to train. It is a fast twitch muscle which means that you have to train it with more intensity. You must fully stretch the calves at the bottom portion of the repetition and contract them in the full range of motion. It is recommended that you hold the contraction for a few seconds to really stimulate all of the calf muscle tissues. There will be a build up of lactic acid as you perform the repetition, however this is the point where you must do a few more reps to get results. Remember, train your calves like they are any other body part, do not put them aside."
+
+  HAMSTRINGS =
+  "The hamstring muscle is located directly at the back of your thigh and works in unison with your quadriceps as well as your glutes and calves. This promotes proper movement of the entire thigh muscles. It is actually composed of three posterior thigh muscles of the semitendinosus, semimembranosus and the biceps femoris.
+
+  The hamstrings muscles play a large role in flexibility of the lower body. Most effective exercises generally are curls at the knee as well a dead-lift type movements. This large muscle group should not be neglected as it will improve mobility and long term flexibility."
+
+  QUADS =
+  "The quadriceps femoris muscle is composed of four prevailing muscles located at the front of the thigh. These four muscle regions are the Rectus femoris, Vastus lateralis, Vastus medialis and finally the Vastus intermedius. The strongest muscle in the body, the quadriceps is responsible for the extension of the knee.
+
+  There are several leg exercise that successfully target the quads. The most effective of the bunch tend to be the leg press as well as the squat. One excellent isolation exercise is the leg extension."
+
+  GLUTES =
+  "Gluteus maximus, also commonly referred to as the glutes, this large and strong muscle group located on your buttocks is the most superficial of the group. It is responsible for many movements originating from the trunk as well as thigh movements.
+
+  The glutes make up a large portion and therefore the overall shape of your backside. Many women as well as men aim to tone this muscle group for a thinner more appealing physique. Many thigh exercises will target the glutes such as lunges and squats. There are however many glute isolating exercises as well such as step-ups and hip extensions."
+
+  PECS =
+  "A thick and fan-shaped muscle, the pectoralis major is essentially the main component of a male chest and lies beneath the breast tissue in women. Below this muscle lies the pecoralis minor. Responsible for the movement of the shoulder joint primarily, the pectoralis major has four major actions.
```

Comment:
Line is too long. [300/80]<br>Indent the first line of the right-hand-side of a multi-line assignment.<br>Freeze mutable objects assigned to constants.